src/Driver/Driver.jl

```diff
@@ -307,9 +307,9 @@ function parse_commandline(
         arg_type = NTuple{2, Int}
         default = get_setting(:degree, defaults, global_defaults)
         "--nelems"
-        help = "tuple of number of elements in each direction: 2 for GCM or 1 for single-stack (no space before/after comma)"
-        metavar = "<nelem_1>,<nelem_2>"
-        arg_type = NTuple{2, Int}
+        help = "tuple of number of elements in each direction: 3 for Ocean, 2 for GCM or 1 for single-stack (no space before/after comma)"
```

Comment:
This is getting a bit verbose. Maybe remove "tuple of" and the part in parentheses (here and below)? Also, should be "Atmos GCM" and "Atmos single-stack".

Reply:
Thanks @kpamnany , I will simplify and specify Atmos for GCM and single-stack. I thought it was clear b/c those are the only one we have. But if one day our suite of configs could be expanded (perhaps there could be an Ocean GCM? I don't know) I will make it clear. 

Regarding the text in parentheses, I thought we said last time that we wanted to document it for the user so that bash doesn't get angry at them, if they accidentally use it with a space in between

Reply:
Okay, I think I took care of it in [
d02b9a1](https://github.com/CliMA/ClimateMachine.jl/pull/1960/commits/d02b9a1630b9f4307ae31eafc3030e9389966025). If it looks good, I will squash a few review commits and merge after CI passes. Thanks